apps/t_shirt_delivery/views.py

```diff
@@ -89,6 +90,14 @@ def get_context_data(self, **kwargs):
             )
         except DeliveryBatchDeadline.DoesNotExist:
             pass
+
+        context["all_tshirts"] = serializers.serialize(
+            "json",
+            TShirtSize.objects.filter(
+                campaign=self.user_attendance.campaign, available=True
+            ),
+        )
```

Comment:
For some reason black doesn't like this. Can you run `./runblack.sh` please, to fix formatting?